Add API explanation to README (#1337)
w3c/specberus: README.md

The special profile `auto` is also available.

### Examples
#### 1. Get API version of Pubrules

`https://www.w3.org/pubrules/api/version`

e.g. https://www.w3.org/pubrules/api/version

#### 2. Get metadata of one document.

`https://www.w3.org/pubrules/api/metadata?url=https://example.com/doc.html`

> Metadata is a bunch of data extracted from the document. It includes type (profile) of the document, publish date, editor(s)' name(s), Patent Policy version the document is under, etc...

Reviewer: ```suggestion
Metadata is a bunch of data extracted from the document. It includes the type (profile) of the document, publish date, editors' names, Patent Policy version the document is under, etc...
```

e.g. https://www.w3.org/pubrules/api/metadata?url=https://www.w3.org/TR/2021/WD-i18n-glossary-20210708/

#### 3. Validate the document using profile: auto

`https://www.w3.org/pubrules/api/validate?url=https://example.com/doc.html&profile=auto`

> `auto` profile is the easiest way to validate an document. The validation relays on the automatically extracted data.

Reviewer: ```suggestion
`auto` profile is the easiest way to validate a document. The validation relies on the automatically extracted data.
```

> The validation result contains both the **metadata** and the **errors** and **warnings** regarding the document.

Reviewer: ```suggestion
The validation result contains both the **metadata** and the **errors**/**warnings** regarding the document.
```

e.g. https://www.w3.org/pubrules/api/validate?url=https://www.w3.org/TR/2021/WD-i18n-glossary-20210708/&profile=auto

#### 4. Validate the document using manual configs

> 

Reviewer: ```suggestion
```

`https://www.w3.org/pubrules/api/validate?url=https://example.com/doc.html&profile=WD&validation=simple-validation&patentPolicy=pp2020`

> Pubrules support advanced configs to make the validation more accurate.

Reviewer: ```suggestion
Pubrules supports advanced configs to make the validation more accurate.
```
